Src/BootCamp.Chapter/Demo1.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    static class Demo1
+    {
+        public static void Demo()
+        {
+            var Manager = new CredentialsManager(""); 
+            
+            Console.WriteLine("Would you like to register or login");
+
+            Console.WriteLine("Choose R for register");
```

Comment:
Instead of R and L use your constants here as well.

Reply:
changed